js/src/common/Application.js

```diff
@@ -331,19 +331,31 @@ export default class Application {
       }
 
       const isDebug = app.forum.attribute('debug');
+      // contains a formatted errors if possible, response must be an JSON API array of errors
+      // the details property is decoded to transform escaped characters such as '\n'
+      const formattedError = error.response && Array.isArray(error.response.errors) && error.response.errors.map(e => decodeURI(e.detail));
 
       error.alert = new Alert({
         type: 'error',
         children,
         controls: isDebug && [
-          <Button className="Button Button--link" onclick={this.showDebug.bind(this, error)}>Debug</Button>
+          <Button className="Button Button--link" onclick={this.showDebug.bind(this, error, formattedError)}>Debug</Button>
         ]
       });
 
       try {
         options.errorHandler(error);
       } catch (error) {
+        if (isDebug) {
+          console.group(`${error.options.method} ${error.options.url} ${error.xhr ? error.xhr.status : ''}`);
+
+          console.error(...formattedError || [error]);
```

Comment:
Is `error` inside an array because the `...` operator is evaluated last?

In other words, is this `...(formattedError || [error])` or `(...formattedError) || [error]`?

Reply:
After some quick testing, this looks to be `...(formattedError || [error])`. Right now, `(...formattedError) || error` doesn't to seem valid (thinks it's an arrow function).

![image](https://user-images.githubusercontent.com/6401250/70376960-9468f680-18dc-11ea-8cb3-f3911a829a21.png)